Intern/rayx-core/src/Shader/Refrac.h

```diff
@@ -6,24 +6,13 @@
 /**
 calculates refracted ray
 @params: 	r: ray
-            normal: normal at intersection point of ray and element
+            normal: normal at intersection point of ray and element -> for planes normal is always the same (0,1,0) -> no rotation and thus no trigonometric
```

Comment:
This comment is a bit confusing. The function does not check the normal values. The behavior is not different for a normal of (0,1,0)